SerhatTan/hw5/yemek-sepeti-frontend/controllers/restaurantsController.js

```diff
@@ -0,0 +1,13 @@
+angular.module('foodDeliveryApp')
+.controller('RestaurantController', restaurantController);
+
+function restaurantController($scope, DataService) {
+	DataService.getRestaurants(function(response) {
+		$scope.restaurants = response;
+	});
+
+	$scope.getFood = function(id) {
+		window.location = "food.html?val="+id;
```

Comment:
Burada food ve restaurant html i iki ayrı dosya olarak tasarladığım için restaurant.html de restaurantlardan birine tıklandığında o restaurantın id sini alıp foodlarını listeleyebilmek için restaurant id sini url aracılığıyla food.html e gönderiyorum.